### PR TITLE
Tests for Cache controller

### PR DESCRIPTION
**Pull request by juliantoledo**
Test for Cache Controller

### CODE REVIEW

File: lib/pwa.js [hunk 1]
```diff
@@ -198,8 +198,12 @@ exports.createOrUpdatePwa = function(pwa) {
       }
       // Delete modified PWA from cache
       const url = '/pwas/' + pwa.id;
-      cache.del(url);
-      cache.del(url + '?contentOnly=true');
+      cache.del(url).catch(_ => {
+        // URL not in cache.
```
Comment (andreban) [inline lib/pwa.js]: Should the catch handler do something, such as logging? Otherwise, we can just remove the catch block.

File: lib/pwa.js [hunk 2]
```diff
@@ -198,8 +198,12 @@ exports.createOrUpdatePwa = function(pwa) {
...
+      });
+      cache.del(url + '?contentOnly=true').catch(_ => {
+        // URL not in cache.
```
Comment (andreban) [inline lib/pwa.js]: See previous comment.

File: lib/pwa.js (hunk 1 above)
Comment (juliantoledo) [inline lib/pwa.js]: done, I removed both. I was trying to get rid of 2 promise unhandled warnings on the test, but this did not help.

File: lib/pwa.js (hunk 2 above)
Comment (juliantoledo) [inline lib/pwa.js]: done

**Comment by juliantoledo** [comment]
@andreban I just removed the two catch blocks.

**Pull request merged**